CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # PayPal Android SDK Release Notes
 
+## unreleased
```

Comment:
Q: Will all these entries get added to release notes when releasing new version of SDK or we need to manually add

Reply:
Yes definitely. For now, they'll accumulate on the `v3` branch. Eventually, they'll get added to the `main` branch release notes once we make it generally available.